tests/functional/a/arguments_differ.py

```diff
@@ -27,7 +27,7 @@ def test(self, arg=None): # [arguments-differ]
 class Classmethod(object):
 
     @classmethod
-    def func(cls, data):
+    def func(cls, data: str):

```

Comment:
Sorry for not being very clear ;) For example here we should keep this change (adding typing) for step 2.
```suggestion
    def func(cls, data):
```

Reply:
(No change for python files in ``tests/functional/``)

Reply:
Yes I think typing is necessary if we want to check if the changes we did here actually work, that's why I added it here as well.

Reply:
Fair enough ! :)